src/main/java/duke/DeleteCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+/**
+ * Encapsulate the command that allow users to delete existing task from the TaskList,
+ * which is-a Command.
+ */
+public class DeleteCommand extends Command {
+
+    private int taskNumber;
```

Comment:
```suggestion
    private final int taskNumber;
```
Suggesting the addition of `final` modifier.

Reply:
I did not implement it as final as I may need to modify the said variable later on.